packages/design-system/src/Radio/index.tsx

```diff
@@ -56,10 +56,7 @@ export const Radio = styled.label<{
   font-weight: ${typography.p1.fontWeight};
   line-height: ${typography.p1.lineHeight}px;
   letter-spacing: ${typography.p1.letterSpacing}px;
-  color: ${(props) =>
-    props.disabled
```

Comment:
why disabled style is removed?

Reply:
I'm just reverting the changes I made in that commit. None of these changes were on release in the first place.

Reply:
Ok